services/distribution/src/test/java/app/coronawarn/server/services/distribution/objectstore/publish/LocalFileTest.java

```diff
@@ -2,22 +2,22 @@
 
 package app.coronawarn.server.services.distribution.objectstore.publish;
 
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 import java.nio.file.Path;
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.ValueSource;
 
 class LocalFileTest {
+
   @ParameterizedTest
   @ValueSource(strings = {
       "version",
       "version/v1/configuration/country",
       "version/v1/diagnosis-keys/country",
       "version/v1/diagnosis-keys/country/DE/date",
-      "version/v1/diagnosis-keys/country/DE/date/2020-06-11/hour" })
+      "version/v1/diagnosis-keys/country/DE/date/2020-06-11/hour"})
```

Comment:
Seems that your IDE does some format changes here. Not too important, but can be a real pain for bigger PRs.

Reply:
I think there was a space that was deleted when I have reformatted the code 🤔